content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -283,6 +283,7 @@ Jobs with _fixed completion count_ - that is, jobs that have non null
 - `Indexed`: the Pods of a Job get an associated completion index from 0 to
   `.spec.completions-1`. The index is available through three mechanisms:
   - The Pod annotation `batch.kubernetes.io/job-completion-index`.
+  - The pod label `batch.kubernetes.io/job-completion-index` (for v1.28 and later).
```

Comment:
Please capitalize “pod” consistently over these 2 lines and ideally over the entire list.